packages/js_of_ocaml-webidl/js_of_ocaml-webidl.0.2/opam

```diff
@@ -9,7 +9,7 @@ doc: "https://tari3x.github.io/webgpu"
 bug-reports: "https://github.com/tari3x/webgpu"
 depends: [
   "dune" {>= "2.2"}
-  "ocaml" {>= "4.08"}
+  "ocaml" {>= "4.08" & < "4.13"}
```

Comment:
I’m not really sure about this. I think it comes from something else (menhir it looks like).

See http://check.ocamllabs.io/log/1647924749-582c5d3aa43a46e397334b9605d6492ccbeaa156/4.14/good/js_of_ocaml-webidl.0.2 where it compiles fine even up to 4.14

Reply:
Oh interesting. I went to look into it. The bounds on menhir for the previous versions of webidl needed to be updated (and in one case actually added). Thanks for the link